test/lib/network/edit_nic_name_test.rb

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env rspec
+
+require_relative "../../test_helper"
+
+require "yast"
+require "network/edit_nic_name"
+
+current_name = "spec0".freeze
```

Comment:
Leftovers? Or did you want to use them as the value for below `let`s? In such case, `current_name` and `new_name` are not being used.

Reply:
completely leftovers :)